docs/pages/agents/build-an-agent.mdx

```diff
@@ -100,7 +100,38 @@ To learn more, see the example [xmtp-attachments](https://github.com/ephemeraHQ/
 
 If helpful, you can also see the [Support attachments](/inboxes/content-types/attachments) documentation. 
 
-### Try out your agent using xmtp.chat
+### Support replies
+
+To learn more, see the example [good example for this?](#TODO) agent.
+
+If helpful, you can also see the [Support replies](/inboxes/content-types/replies) documentation. 
+
+### Support reactions
+
+To learn more, see the example [good example for this?](#TODO) agent.
+
+If helpful, you can also see the [Support reactions](/inboxes/content-types/reactions) documentation. 
+
+### Follow security best practices
+
+For an agent to function—whether it's answering questions, executing commands, or providing automated responses—it must be able to read the conversation to understand what's being asked, and write messages to respond. 
+
+Like any other user, this means your agent holds the cryptographic keys required to decrypt and send messages in the conversation. As an agent developer, it's important to uphold the security of these keys and messages.
+
+Here are some security best practices:
+
+- **Keep keys and messages secure and private**: Do not log messages in plaintext. Do not share messages with third parties.
+- **Label agents clearly**: Clearly identify your agent as an agent and don't have an agent impersonate a human.
+
+### Handle rate limits
```

Comment:
Line item from Base docs: Rate Limiting: Respect XMTP rate limits in your agent logic

Do we want to say anything more specific?